sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOperation.cs

```diff
@@ -101,14 +118,20 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     ? await _serviceClient.GetAnalyzeLayoutResultAsync(new Guid(Id), cancellationToken).ConfigureAwait(false)
                     : _serviceClient.GetAnalyzeLayoutResult(new Guid(Id), cancellationToken);
 
-                if (update.Value.Status == OperationStatus.Succeeded || update.Value.Status == OperationStatus.Failed)
+                _response = update.GetRawResponse();
+
+                if (update.Value.Status == OperationStatus.Succeeded)
                 {
                     _hasCompleted = true;
```

Comment:
Per @kinelski's comment, let's switch the order of `_hasCompleted` and `_value= ...` here and elsewhere.